scripts/zones/Ship_bound_for_Selbina/npcs/Bhagirath.lua

```diff
@@ -21,7 +21,33 @@ end;
 -----------------------------------
 
 function onTrigger(player,npc)
-	player:messageSpecial(ON_WAY_TO_SELBINA,0,0); -- Earth Time, Vana Hours. Needs a get-time function for boat?
+
+	local vHour = VanadielHour();
+	local vMin  = VanadielMinute();
+
+	while vHour >= 6 do
+		vHour = vHour - 8;
+	end
+
+	if(     vHour == -2) then vHour = 8;
```

Comment:
Are you trying to vertically align your var names or "then"s or something?
```lua
if (vHour == -2) then
    vHour = 8;
elseif (vHour == -1) then
    vHour = 7;
elseif (vHour == 0) then
    vHour = 6;
elseif (vHour == 1) then
    vHour = 5;
elseif (vHour == 2) then
    vHour = 4;
elseif (vHour == 3) then
    vHour = 3;
elseif (vHour == 4) then
    vHour = 2;
elseif (vHour == 5) then
    vHour = 1;
end
```
This is the style usually used, though not strictly enforced (I wish it were).